app/components/map/map_component.jsx

```diff
@@ -114,7 +114,8 @@ const MapComponent = ({ zoomLevel, navigation}) => {
 
         ))}
 
-      <ToolbarComponent style={mapStyles.navigation} navigation={navigation}></ToolbarComponent>
+      <ToolbarComponent //style={mapStyles.navigation} 
+      navigation={navigation}></ToolbarComponent>
```

Comment:
(PR review assignment:)
The indentation is not consistent.
Do you have any linter?